scripts/communication.js

```diff
@@ -3,26 +3,54 @@
 //   If somebody in chat write appropriate query,
 //   mona will answer him a random response for this query.
 
-const greetings = ['Hello', 'Good morning', 'G\'day', '\^Hi\$', 
-  'Доброго утра', 'Доброе утро', 'Привет'];
+const incomingGreetings = [
+  'Hello',
+  'Good morning',
+  'G\'day',
+  '\^Hi\$',
+  'Morning',
+  'Hey',
+  'Wazzzup',
+];
+
+const responsesOnName = [
+  'Whazzzup bro?',
+  'How can I help?',
+  'Yeap?)',
+  'Mm?)',
+  "I'm here)",
+];
+
+const responsesOnGreeting = [
+  'Hey, dude!',
+  'Whatzzzup, bro :smiley:',
+  'Morning, bro :smiley:',
+  'Nice... :smiley:',
+  'Hello! :smiley:',
+  'Morning, bro :smiley:',
+  'Hi, how are you?',
+  'Hell ou',
+];
+
+const cyrillicQuery = '[\u0400-\u04FF]+';
 
 const handlerCommunication = (robot, queries, answers) => {
   queries.forEach((query) => {
     robot.hear(new RegExp(query, 'i'), (response) => {
-      response.send(response.random(answers));
+      if (query === cyrillicQuery) {
+        return response.send('Dude, uncool. Speak in English, please :slightly_smiling_face:');
+      }
```

Comment:
Will it be called on each Russian letter in every channel?

Reply:
🤔

Reply:
@DDKatch am I missing something here?

Reply:
@AleksSenkou looks like it's impossible to test it correctly in local environment, so I'm going to remove this code from here

Reply:
will open an issue for that with some code tips